### DISCUSSION TITLE
第三者レビュー メッセージ-ソースコード

### DISCUSSION THREAD

**Discussion by creature-water-valley** [ws-4020/mobile-app-crib-notes#573] [General]
### ソースコード

- メッセージキーが見つからない場合はメッセージキーを返却していますが、処理続行できるようにしているのでエラーレベルが警告ではないのか少し気になりました

- プレースホルダーで例えば`{0}`を同じメッセージ中に複数定義された場合の挙動が気になりました。方式の「メッセージのロード」では触れられておらず、実装を見ると最初にHitしたものが置換されるように見えましたので、仕様としてどこかに記載なくてもいいのか少し気になりました。

**Comment by creature-water-valley** [comment]
### 対応内容

- Firebase Crashlytics でエラーを収集しているから対応しない
- 正規表現を用いた置換でコードを修正する

**Comment by kozake** [comment]
### 対応PR

https://github.com/ws-4020/mobile-app-crib-notes/pull/560

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
